test/util/bitcoin-util-test.py

```diff
@@ -77,32 +77,36 @@ def bctest(testDir, testObj, buildenv):
     are not as expected. Error is caught by bctester() and reported.
     """
     # Get the exec names and arguments
-    execprog = buildenv["BUILDDIR"] + "/src/" + testObj['exec'] + buildenv["EXEEXT"]
+    execprog = os.path.join(buildenv["BUILDDIR"], "src", testObj["exec"]) + buildenv["EXEEXT"]
```

Comment:
I think this is better as:

```python
execprog = os.path.join(buildenv["BUILDDIR"], "src", testObj["exec"] + buildenv["EXEEXT"])
```

(ie `buildenv["EXEEXT"]` should be appended to `testObj["exec"]` before joining into a path), but I don't have a windows machine to test this on.

Is there a reason you put `buildenv["EXEEXT"]` outside the `os.path.join`?

Reply:
I thought also but that was mistook, buildenv["EXEEXT"] is not path, so occur error when with os.path.join
that need just string concatenation

![comment](http://i.imgur.com/ShpEvJv.png)

Reply:
right, that's why you should have:

```
execprog = os.path.join(buildenv["BUILDDIR"], "src", testObj["exec"] + buildenv["EXEEXT"])
```

(concatenate `testObj["exec"]` and  `buildenv["EXEEXT"]` as a single argument to `os.path.join()`)

Reply:
oh, sorry I misunderstood, that's look better, I'll update :)